src/CMakeLists.txt

```diff
@@ -74,29 +85,37 @@ list(APPEND SOURCES
   physics/em/MuBremsstrahlungModel.cc
   physics/em/RayleighModel.cc
   physics/em/RayleighProcess.cc
-  physics/em/detail/Rayleigh.cc
+  physics/em/detail/RayleighData.cc
+  physics/em/generated/BetheHeitlerInteract.cc
+  physics/em/generated/EPlusGGInteract.cc
+  physics/em/generated/KleinNishinaInteract.cc
+  physics/em/generated/LivermorePEInteract.cc
+  physics/em/generated/MollerBhabhaInteract.cc
+  physics/em/generated/MuBremsstrahlungInteract.cc
+  physics/em/generated/RayleighInteract.cc
+  physics/em/generated/SeltzerBergerInteract.cc
```

Comment:
Just as a TODO here: we should have the `celeritas_gen_interactor` script return the name of source files it created, so we can just add `${_GEN_SOURCES}` to this list.

Reply:
I went ahead and implemented this. CMake is funky as you know with functions not being able to return values -- let me know if what I did is not what you had in mind